docker-compose-starburst.yml

```diff
@@ -16,6 +16,37 @@ services:
       POSTGRES_USER: dbt-trino
```

Comment:
Can you please disable running postgres container from docker-compose? See https://stackoverflow.com/a/65957695